apps/api/src/server/applications/application/documents/document.routes.js

```diff
@@ -593,18 +592,13 @@ router.get(
         #swagger.responses[200] = {
 			description: 'An paginated data set of matching documents and their properties',
             schema: { $ref: '#/definitions/PaginatedDocumentDetails' }
-        }
-		#swagger.responses[400] = {
-            description: 'Search criteria too short',
```

Comment:
don't forget to rebuild the swagger docs, as the definition has changed here.

@lutamKC also I have spotted an existing error in swagger.js - the PaginatedDocumentDetails obj (which we are also using in this route) is wrongly declared to be an array, should just be an object - swagger.js line 253

Reply:
ok found it, I think now it's correct

Reply:
nice one, all looks good!